.github/workflows/merge-release-branch.yml

```diff
@@ -0,0 +1,22 @@
+name: Merge Release Into Master
+
+on: repository_dispatch
+
+jobs:
+  merge_to_master:
+    runs-on: ubuntu-latest
+    # Allow GITHUB_TOKEN to have write permissions
+    permissions:
+      contents: write
+    steps:
+      - name: Merge to master
+        uses: actions/github-script@0.9.0
```

Comment:
Looks like the latest version is v6 now (includes breaking changes). Worth checking if we can use one of the recent versions here.